### PR TITLE
NPM Workflow + Add components to be used in Cloud

### PR DESCRIPTION
**Pull request by madebysid**
First off, sorry about this absolute unit of a PR. In the future I will never send you a huge PR like this.

I'm adding people as reviewers, but I do not expect any of you to review the whole thing. Since these components are only used in Cloud, we should be fine.

Blocked by:
- [x] https://github.com/prisma/docs/pull/1459
- [x] https://github.com/prisma/prisma-blog/pull/199

NPM Package: https://www.npmjs.com/package/@prisma/lens

New components: (https://6033a0d2b7016200231f4cce-quqrrepdtp.chromatic.com/)
- Button
- Label
- TextField
- Select
- ComboBox
- Menu
- Dialog
- Table
- FocusRing
- Form
- Icon
- PageTitle
- Card
- InfoCard
- Separator

Also added a few showcase stories (Preview on Chromatic)

I eyeballed/guessed styles since I couldn't find all states in the designs (I've been referencing https://www.figma.com/file/ERryxBnIxYkXKso5hrQb20/Prisma-Cloud-Prototype?node-id=184%3A2), but I think Kuldar/Matej will be ablet o change things pretty easily.

Also closes https://github.com/prisma/lens/issues/26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +27,49 @@ Should loosely follow [the theme spec](https://system-ui.com/theme) with followi
...
+
+To publish manually, you should:
+
+```
+npm login # Login to NPM
+npm version patch # Bump up the version
+npm publish # Publish package
+```
+
+## Usage in other Prisma projects
```
Comment (madebysid) [inline README.md:54]: @hrescak This should be what we discussed on Friday

**Comment by madebysid** [comment]
Important things to know about the setup:

- The library has two entrypoints currently:
  - `./lib.ts`: For all components. This is used by Cloud
  - `./web.ts`: For website components. This is used by `homepage-v8`, `docs` & `prisma-blog`.
  
- Each entrypoint exposes a `LensProvider` that imports styles, fonts and other required stuff. This way, the consumer does not have to worry about importing these things.
  - It loads styles by injecting them on page load. Not ideal since they are now deferred till JS loads, but you don't see content till JS loads anyway, so fine I guess
  - It loads fonts by base64 encoding them.
  - The `lib.ts` Provider also sets up SSR for `react-aria`.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": false
+}
```
Comment (jasonkuhrt) [inline .prettierrc:6]: Should have a prisma config that we inherit from like we did on Prisma labs?

File: package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
   "homepage": "https://github.com/prisma/lens#readme",
   "scripts": {
     "dev": "start-storybook -p 6006 -s public --ci",
+    "format": "prettier -w . --ignore-path .gitignore",
```
Comment (jasonkuhrt) [inline package.json:32]: Oh nice, so we don't need to have a `.prettierignore` then? TIL

**Review by jasonkuhrt — APPROVED** [review]
LGTM! 📦

**Comment by madebysid** [comment]
yolo

File: .prettierrc (hunk 2 above)
Comment (madebysid) [reply .prettierrc:6]: Might be overkill IMO 😅 
We can if everyone wants though

**Pull request merged**